erp-data-exchange-specification.md

```diff
@@ -154,63 +156,111 @@ The `utilisationPeriods.csv` describes utilisation periods.
+The `tenants.csv` file describes a tenant.
 
 #### Fields
 
-| Field                   | Type                        | Description                                                                                           |
-| ----------------------- | --------------------------- | ----------------------------------------------------------------------------------------------------- |
-| **importType**          | [Import Type](#import-type) | Must always use "insert"
-| **utilisationPeriodId** | [UUID](#uuid-type)          | The foreign UUID of the utilisation period the tenant check-in is for (_utilisationPeriods.csv_ `id`) |
-| **tenantId**            | [UUID](#uuid-type)          | The foreign UUID of the tenant tenant check-in is for (_tenantss.csv_ `id`)                           |
+| Field                | Type                        | Description              |
+| -------------------- | --------------------------- | ------------------------ |
+| **importType**       | [Import Type](#import-type) | One of:<br/>`insert`, `update` 
+| **residentId**               | [UUID](#uuid-type)          | Your UUID for the tenant-user |
+| **name** | [string](#string-type)      | The tenant's name
+| **email** | [Email](#string-type)      | The tenant's email address
+
+#### Example
+
+```csv
+importType,residentId,name, email
+insert,db8b732f-e0ff-41d9-9c15-ca1be2776fd4, Harry Potter. harrypotter@hogwards.edu
+insert,ad257d42-1078-4279-9918-e774859555ae,Dolores Umbridge, doloresumbridge@hogwards.edu
+insert,2f0fc4c6-3a5e-4cd7-9007-9add50653be5,Severus Snape, severus.snape@hogwards.edu
+insert,41f93044-e050-4a33-9e52-ebdada55f9a7,Albus Dumbledore, albus-d@hogwards.edu
+```
+
+### companies.csv
+
+The `companies.csv` file describes companies.
+
+#### Fields
+
+| Field                | Type                        | Description              |
+| -------------------- | --------------------------- | ------------------------ |
+| **importType**       | [Import Type](#import-type) | One of:<br/>`insert`, `update` 
+| **id**               | [UUID](#uuid-type)          | Your UUID for the company |
+| **name** | [string](#string-type)      | The company name |
+| **email** | [Email](#string-type)      | The company agents email address
 
 #### Example
 
 ```csv
-importType,tenantId,utilisationPeriodId
-insert,7d87a383-1778-401b-a421-b60c900479c3,db8b732f-e0ff-41d9-9c15-ca1be2776fd4
-insert,7d87a383-1778-401b-a421-b60c900479c3,db8b732f-e0ff-41d9-9c15-ca1be2776fd4
-insert,7d87a383-1778-401b-a421-b60c900479c3,ad257d42-1078-4279-9918-e774859555ae
-insert,7d87a383-1778-401b-a421-b60c900479c3,ad257d42-1078-4279-9918-e774859555ae
+importType,residentId,name
+insert,db8b732f-e0ff-41d9-9c15-ca1be2776fd4,Big Company, Inc., info@bigcomp.com
+insert,ad257d42-1078-4279-9918-e774859555ae,Auto GmbH, ceo@autogmbh.com
+insert,2f0fc4c6-3a5e-4cd7-9007-9add50653be5,Fuji LLC, boss@fuji.com
+insert,41f93044-e050-4a33-9e52-ebdada55f9a7,West & Smith, Inc., manager@ws.com
```

Comment:
Strings that include commas should be enclosed in " 
like:
insert,41f93044-e050-4a33-9e52-ebdada55f9a7,"West & Smith, Inc.", manager@ws.com